mms/serving_frontend.py

```diff
@@ -370,8 +370,8 @@ def ping_callback(self, **kwargs):
             for model in self.service_manager.get_loaded_modelservices().values():
                 model.ping()
         except Exception:
-            if 'error_metric' in MetricsManager.metrics:
-                MetricsManager.metrics['error_metric'].update(metric=1)
+            if '500error_metric' in MetricsManager.metrics:
```

Comment:
Can we call this 5XX and not 500? Same for 4XX vs 400.

Reply:
changed.